pkg/piecestore/rpc/client/api/api.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package api
```

Comment:
Does it really make sense to have "rpc/client/api" and "rpc/server/api" packages both ending with "api". Why not just "rpc/client" and "rpc/server"?

Reply:
:+1:

Reply:
@aleitner